dev/io.openliberty.checkpoint_fat_transaction/publish/servers/checkpointTransactionServlet/server.xml

```diff
@@ -0,0 +1,67 @@
+<!--
+    Copyright (c) 2023 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License 2.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-2.0/
+
+    SPDX-License-Identifier: EPL-2.0
+
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+
+<server>
+    <include location="../fatTestPorts.xml"/>
+
+    <featureManager>
+      <feature>checkpoint-1.0</feature>
+      <feature>servlet-4.0</feature>
+      <feature>jndi-1.0</feature>
+      <feature>ejbLite-3.2</feature>
+      <feature>jdbc-4.2</feature>
+      <feature>txtest-1.0</feature>
+      <feature>componentTest-1.0</feature>
+    </featureManager>
+
+    <sslDefault outboundSSLRef="mySSLConfig" />
+
+    <!-- Server SSL configuration -->
+    <ssl id="defaultSSLConfig" keyStoreRef="defaultKeyStore" trustStoreRef="defaultTrustStore" /> 
```

Comment:
What feature pulls in ssl support?

Reply:
None of the features pull in ssl-1.0.  Yanked the ssl config and local testing works.  I suspect it's there to facilitate copy-paste in other test servers in the TX FAT.